add ContentStructureFactory (#10)
code-rhapsodie/ezdataflow-bundle: README.md

            $remoteId = sprintf('article-%d', $data['id']);
            unset($data['id']);

            return $this->contentStructureFactory->transform(
                    $data,
                    $remoteId,

>                     $lang,

Reviewer: I think `$lang` is no longer defined.

                    'article2',
                    54 //Parent location id
                );
        });
        $builder->addWriter($this->contentWriter);
    }
}
```

> This example use `ContentStructureFactory` for check if the content exist and return the good `ContentStrucure` for content writer.

Reviewer 2: ```suggestion
This example uses `ContentStructureFactory` to check if the content exists and returns the good `ContentStrucure` to the content writer.
```

> This example uses `ContentStructureFactory` to check if the content exists and returns the good `ContentStrucure` to the content writer.

Reviewer: ```suggestion
This example uses `ContentStructureFactory` to check if the content exists and returns the adequate `ContentStrucure` to pass to the content writer.
```
